src/transport/raw/PeerAddress.h

```diff
@@ -54,6 +54,7 @@ enum class Type : uint8_t
     kBle,
     kTcp,
     kWiFiPAF,
+    kNumofTypes, // This is not an actual transport type, it just refers to the number of transport types
```

Comment:
this means kNumOfTypes will be a valid enum (compiles) but invalid handled anywhere.
How about having `kLast = kWiFiPAF` and then there will never be invalid enums in the code and you also have the count?

Reply:
makes sense, done